### PR TITLE
CODING.md: some tips about refactorings

### PR DESCRIPTION
**Pull request by pedronis**
from recent discussions with team members

### CODE REVIEW

**Review by alfonsosanchezbeato — APPROVED** [review]
Sounds reasonable to me

**Review by ernestl — APPROVED** [review]
Good, thanks

File: CODING.md [hunk 1]
```diff
@@ -209,6 +209,8 @@ The CI tooling will check and enforce the order and required sections when a spr
 
 * Large mechanical refactoring and changes should be done as separate PRs. Try to separate behaviour changes and refactoring into different PRs and not mix the two.
 
+* Refactorings ideally should not touch preexisting tests or only minimally. Before starting a refactor it might be a good idea to check that the coverage (both lines but also feature/behaviors) of the area affected is sufficient to anchor the refactor. Checking the coverage afterward can also reveal whether there is old code that now can be dropped.
```
Comment (degville) [inline CODING.md:212]:
```suggestion
* Refactoring should not touch preexisting tests. If changing a test is unavoidable, changes must be minimal. To ensure a refactor can be anchored, it's good idea to check the coverage before starting, both in terms of lines of codes and in the features and behaviors that may be affected. Checking the coverage afterwards can also reveal whether there is old code that now can be dropped.
```

**Review by degville — APPROVED** [review]
I've made a small suggestion, but this looks good to me too. Thanks for adding this.

**Review by andrewphelpsj — APPROVED** [review]
Thanks!

File: CODING.md (hunk 1 above)
Comment (pedronis) [reply CODING.md]: thanks, applied

**Pull request merged**